### PR TITLE
Destination mysql: upgrade to latest cdk

### PR DESCRIPTION
**Pull request by edgao**

### CODE REVIEW

**Comment by edgao** [comment]
* **#36926** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/36926?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @edgao and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java [hunk 1]
```diff
@@ -133,11 +144,33 @@ protected JdbcSqlGenerator getSqlGenerator() {
     throw new UnsupportedOperationException("mysql does not yet support DV2");
   }
 
+  @Override
+  protected StreamAwareDataTransformer getDataTransformer(ParsedCatalog parsedCatalog, String defaultNamespace) {
```
Comment (edgao) [inline airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java:148]: see MySQLSqlOperations - this is replacing the `formatData` override. (@gisripa doublechecking - am I understanding these two methods correctly?)

**Review by jbfbell — APPROVED** [review]

File: airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java (hunk 1 above)
Comment (gisripa) [reply airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java:148]: Yes correct. your impl of StreamAwareDataTransformer will make it consistent across all.

**Review by gisripa — APPROVED** [review]

File: airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java (hunk 1 above)
Comment (edgao) [reply airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java:148]: oh nice, can I just kill formatData then?

Comment (edgao) [reply airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java:148]: doesn't seem like there's any overrides, I'll remove it
image

Comment (edgao) [reply airbyte-integrations/connectors/destination-mysql/src/main/java/io/airbyte/integrations/destination/mysql/MySQLDestination.java:148]: https://github.com/airbytehq/airbyte/pull/36926/commits/2b3e0f548da1f487db7a16c30b4d477923793954

**Pull request merged**